tools/releasing/create_source_release.sh

```diff
@@ -20,11 +20,13 @@
 # This script relies on few environment variables to determine source code package
 # behavior, those variables are:
 #   RELEASE_VERSION -- The version of this source package.
+#   RELEASE_ROUND -- The round of this version release.
 # For example: RELEASE_VERSION=5.0.0-alpha
+#              RELEASE_ROUND=1
 
 
 RELEASE_VERSION=${RELEASE_VERSION}
-TAG_NAME="v"${RELEASE_VERSION}
+TAG_NAME=${RELEASE_VERSION}(RC${RELEASE_ROUND})
```

Comment:
@wu-sheng I changed it, do you agree?

Reply:
I am OK with this. Let's merge the new doc and script.